apps/authentication/routes.py

```diff
@@ -62,7 +62,7 @@ def login():
         user = Users.query.filter_by(username=username).first()
 
         # Check the password
-        if user and verify_pass(password, user.password):
+        if user and user.password and verify_pass(password, user.password):
```

Comment:
This was a bug where users without a password trying to login would trigger an error.